vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -218,6 +218,33 @@ function Get-ToolsToVHD {
     Expand-Archive -Path "$cacheDir\DU.zip" -DestinationPath "$toolsDir\DU" -Force
 }
 
+function Register-ExpandVolumeTask {
+    $toolsDir = "c:\aks-tools"
```

Comment:
define this in [vhdbuilder/packer/generate-windows-vhd-configuration.ps1](https://github.com/Azure/AgentBaker/pull/4154/files#diff-1a71ca93f12803bce028e28b3600b14d149c348b3c829b2a7fe122b0e017fe25) as @junjiezhang1997 defined `$global:aksTempDir = "c:\akstemp"`?

Reply:
I use the existing folder `c:\aks-tools`, details in line 212. Add this code to decouple function `Get-ToolsToVHD`.
Would you like uses a new folder?

Reply:
I think that we can use the existing tool folder. Suggest to define "c:\aks-tools" as a global variable.

Reply:
Updated.